index.html

```diff
@@ -0,0 +1,29 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <link rel="stylesheet" href="styles.css" />
+    <meta charset="UTF-8" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <title>Document</title>
+  </head>
+  <body>
+    <h1>Overnight Oats</h1>
+    <nav>
+      <a href="Ingredients.html">Ingredients</a>
+    </nav>
+    <div>
```

Comment:
Consider changing this to a `section` element?

Also, for the tabbed UI idea I suggested above, this `section` would then follow with another `section` element with its own heading and the content from lines 15-19 from your `Ingredients.html`